flags/14.0/mysqlctld.txt

```diff
@@ -2,48 +2,33 @@ Usage of mysqlctld:
       --alsologtostderr                                                  log to standard error as well as files
       --app_idle_timeout duration                                        Idle timeout for app connections (default 1m0s)
       --app_pool_size int                                                Size of the connection pool for app connections (default 40)
-      --backup_engine_implementation string                              Specifies which implementation to use for creating new backups (builtin or xtrabackup). Restores will always be done with whichever engine created a given backup. (default builtin)
```

Comment:
Should be pretty much identical to mysqlctl.txt

Reply:
fixed